lib/media/streaming_engine.js

```diff
@@ -1708,6 +1732,54 @@ shaka.media.StreamingEngine = class {
     }
   }
 
+  /**
+   * Parse PRFT box.
+   * @param {!shaka.media.SegmentReference} reference
+   * @param {!shaka.extern.ParsedBox} box
+   * @private
+   */
+  parsePrft_(reference, box) {
+    if (this.parsedPrftEventRaised_) {
+      return;
+    }
+    box.reader.readUint32(); // Ignore referenceTrackId
+    const ntpTimestampSec = box.reader.readUint32();
+    const ntpTimestampFrac = box.reader.readUint32();
+    const ntpTimestamp = ntpTimestampSec * 1000 + ntpTimestampFrac / 2**32;
```

Comment:
(ntpTimestampFrac / 2**32) is fractions of a second.
ntpTimestampFrac / 2**32 * 1000 is ms

Reply:
Good spot this change has been addressed. Thank you!